### PR TITLE
Introduce tests with testinfra tool

### PR DESCRIPTION
**Pull request by renatomefi**

### CODE REVIEW

File: php-fpm-healthcheck [hunk 1]
```diff
@@ -103,10 +103,14 @@ check_fpm_health() {
     done
 }
 
+set +e
+
 GETOPT=$(getopt -o v --long verbose,accepted-conn:,listen-queue-len:,active-processes:,slow-requests: -n 'php-fpm-healthcheck' -- "$@")
 
 if test "$?" != 0; then >&2 echo "Invalid options, terminating." ; exit 3 ; fi
 
+set -e
```
Comment (stevenjm) [inline php-fpm-healthcheck]: This is probably better done as `|| true` after the command that may fail, so it's clear which command's exit status we're ignoring.

File: test/docker.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+declare CONTAINER
+CONTAINER=$(docker run -d --rm "$DOCKER_TAG_TEMPORARY")
+
+declare TESTS_DIR
+TESTS_DIR="$(pwd)/test"
+
+docker run --rm -t \
+		-v "$TESTS_DIR:/tests" \
+		-v /var/run/docker.sock:/var/run/docker.sock:ro \
+		renatomefi/docker-testinfra:latest --verbose --hosts="docker://$CONTAINER"
```
Comment (stevenjm) [inline test/docker.sh]: The indentation here mixes 2 spaces with tabs. We've standardised on 4-space indents for most stuff.

File: test/testinfra/test_command.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+def test_healthcheck_script_is_available(host):
+  cmd = host.run("which php-fpm-healthcheck")
+  assert cmd.rc == 0
+
+def test_healthcheck_script_is_executable(host):
+  scriptFile = host.check_output("which php-fpm-healthcheck")
+    
+  assert host.file(scriptFile).exists is True
+  assert host.file(scriptFile).mode == 0o775
```
Comment (stevenjm) [inline test/testinfra/test_command.py]: Ditto 4-spaced indentation here.

File: test/testinfra/test_ping.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+def test_ping(host):
+  cmd = host.run("php-fpm-healthcheck")
+  assert cmd.rc == 0
+
+def test_ping_verbose(host):
+  cmd = host.run("php-fpm-healthcheck -v")
+  assert cmd.rc == 0
+  assert "Trying to connect to php-fpm via:" in cmd.stdout
+  assert "status output:" in cmd.stdout
+  assert "pool:" in cmd.stdout
```
Comment (stevenjm) [inline test/testinfra/test_ping.py]: Ditto 4-spaced indentation here.

File: php-fpm-healthcheck (hunk 1 above)
Comment (renatomefi) [reply php-fpm-healthcheck]: It keeps the previous command status?

Comment (renatomefi) [reply php-fpm-healthcheck]: fixed!

**Pull request merged**